test/integration/prerender/test/index.test.js

```diff
@@ -576,6 +576,30 @@ const runTests = (dev = false, isEmulatedServerless = false) => {
     expect(value).toMatch(/Hi \[second\]!/)
   })
 
+  it('should not return data for fallback: false and missing dynamic page', async () => {
```

Comment:
We don't want to run this test for emulated serverless mode right?

Reply:
Oops, correct!